app/src/main/res/drawable/amplifying_on_blue.xml

```diff
@@ -0,0 +1,18 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
There's probably a way to parameterize the colors used here and only need one `amplifying_on.xml`, but I'm too lazy/it's not a high enough priority for me to investigate how.

Reply:
too lazy :'(